content.md

```diff
@@ -185,7 +185,7 @@ Tap on form field and type the desired value using the pop-up keyboard.
 
 ### 5.2 Auto LookUp
 
-Start completing the form field and select the applicable item from the list displayed below the text field.
+Start filling in the first 3 letters of the parameter(eg. Member, Garden ect.) you want to add to the form field. Select the relevant item from the list displayed below the text field. If this parameter(eg. Member, Garden ect.) does not appear in the list below the text field it has not been submitted to the Kandu database.
```

Comment:
Please add images that show what the auto lookup should look like when items are listed.